spring-integration-core/src/main/java/org/springframework/integration/config/IntegrationManagementConfigurer.java

```diff
@@ -506,4 +506,8 @@ private void registerComponentGauges() {
 		return null;
 	}
 
+	private ManagementOverrides getOverrides(final IntegrationManagement bean) {
```

Comment:
Should be `static`. No `final` for arg, please.
Your name to the `@author` list.

Thank you!

Reply:
@artembilan Addressed your comments.